modules/virt-about-aaq-operator.adoc

```diff
@@ -0,0 +1,77 @@
+
+
+.AAQ Operator controller and custom resources
+The AAQ Operator introduces two new API objects defined as custom resource definitions (CRDs) for managing alternative quota implementations across multiple namespaces:
+
+* `ApplicationAwareResourceQuota`: Sets aggregate quota restrictions enforced per namespace. The `ApplicationAwareResourceQuota` API is compatible with the native `ResourceQuota` object and shares the same specification and status definitions.
++
+.Example manifest
+[source,yaml]
+----
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareResourceQuota
+metadata:
+  name: example-resource-quota
+spec:
+  hard:
+    requests.memory: 1Gi 
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1" # <1>
+    requests.memory/vmi: 1Gi # <2>
+# ...
+----
+<1> The maximum amount of CPU that is allowed for VM workloads in the default namespace.
+<2> The maximum amount of RAM that is allowed for VM workloads in the default namespace.
+
+* `ApplicationAwareClusterResourceQuota`: Mirrors the `ApplicationAwareResourceQuota` object at a cluster scope. It is compatible with the native `ClusterResourceQuota` API object and shares the same specification and status definitions. AAQ cluster quotas are not namespaced. When creating an AAQ cluster quota, you can select multiple namespaces based on annotation selection, label selection, or both by editing the `spec.selector.labels` or `spec.selector.annotations` fields.
++
+.Example manifest
+[source,yaml]
+----
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareClusterResourceQuota # <1>
+metadata:
+  name: example-resource-quota
+spec:
+  quota:
+    hard:
+    requests.memory: 1Gi
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1"
+    requests.memory/vmi: 1Gi
+  selector:
+    annotations: null
+    labels:
+      matchLabels:
+        kubernetes.io/metadata.name: default
+# ...
+----
+<1> You can only create an `ApplicationAwareClusterResourceQuota` object if the `spec.allowApplicationAwareClusterResourceQuota` field in the `HyperConverged` custom resource (CR) is set to `true`.
++
+[NOTE]
+====
+If both `spec.selector.labels` and `spec.selector.annotations` fields are set, only namespaces that match both labels are selected.
+====
+
+The AAQ controller uses a scheduling gate mechanism to evaluate whether there is enough of a resource available to run a workload. If so, the scheduling gate is removed from the pod and it is considered ready for scheduling. The quota usage status is updated to indicate how much of the quota is used. If the CPU and memory requests and limits for the workload exceed the enforced quota usage limit, the pod is not scheduled and remains in `SchedulingGated` status until there is enough quota available.
```

Comment:
The scheduling gate mechanism is new and I'm not sure how many people know about it.
Do we have a reference to it anywhere?

Reply:
What about this [API reference doc](https://docs.openshift.com/container-platform/4.17/rest_api/workloads_apis/pod-v1.html#spec-schedulinggates)? 

The OpenShift Container Platform documentation has an entire [section on pod scheduling](https://docs.openshift.com/container-platform/4.17/nodes/scheduling/nodes-scheduler-about.html) but there is no mention of scheduling gates.

Reply:
@Barakmor1 Any thoughts on which doc to link to?

Reply:
[is](https://docs.openshift.com/container-platform/4.17/rest_api/workloads_apis/pod-v1.html#spec-schedulinggates) is better, in u/s docs we used official k8s docs as reference but i'm not sure if we are allowed to do it here

Reply:
Sounds good. Yeah, we don't typically link to upstream docs as reference.

Reply:
The PR looks good to me, just missing this link.

Reply:
I have added the link in the "Additional resources" section. Unfortunately, due to tooling limitations we cannot add cross references in a module type file.

Here's the latest preview: https://83412--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-understanding-aaq-operator.html#virt-configuring-aaq-operator_virt-understanding-aaq-operator

Reply:
I couldnt find it in the "Additional resources" section.

Reply:
It's the 5th item in the list. Should I change the link text to "Pod scheduling gate specification"?

Reply:
See https://github.com/openshift/openshift-docs/pull/83412#discussion_r1806429284